contrib/win32/win32compat/misc.c

```diff
@@ -1046,8 +1048,228 @@ readpassphrase(const char *prompt, char *outBuf, size_t outBufLen, int flags)
 	return outBuf;
 }
 
-void invalid_parameter_handler(const wchar_t* expression, const wchar_t* function, const wchar_t* file, unsigned int line, uintptr_t pReserved)
+void 
+invalid_parameter_handler(const wchar_t* expression, const wchar_t* function, const wchar_t* file, unsigned int line, uintptr_t pReserved)
 {	
 	debug3("Invalid parameter in function: %ls. File: %ls Line: %d.", function, file, line);
 	debug3("Expression: %s", expression);
 }
+
+int
```

Comment:
make it static.

Reply:
defined in misc_internals.h so that it can be used everywehere, when it is necessary.
So didn't make it as static.